test_crates/generic_type_param_maybe_sized/src/lib.rs

```diff
@@ -0,0 +1,43 @@
+use std::boxed::Box;
+pub struct GenericStruct<T: ?core::marker::Sized, U: core::marker::Sized, V> {
```

Comment:
would you like to add some tests to include traits like `UserSafeSized`(appeared in the `rust` repo) that ends with `Sized`, I believe your code works since trait-bound modifier only work for `Sized` trait